source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md

```diff
@@ -65,3 +65,11 @@ You can destroy the tunnel by using the port value found within the **Connection
 ```bash
 ps -fU USERNAME | grep "ssh -f" | grep "PORT:" | awk '{print $2}' | xargs kill
 ```
+
+## Troubleshooting
+### Lost Connection to MySQL Server

```

Comment:
I think "Lost Connection.." as heading is good. 

Not a big deal, but also wondering about word choice of "idled." Thinking of an idle car...it's still on. Terminus command is "wake" so I'm wondering if describing DB servers as "sleeping" or "hibernating" might make more sense? <shrugs>